### PR TITLE
Lowercase some more forgotten strings

### PR DESCRIPTION
**Pull request by reosarevok**
These were left behind because I guess my original regex didn't play well with `N_l` calls.

### CODE REVIEW

**Review by yvanzo — COMMENTED** [review]
In most cases the context should probably be added.

File: root/entity/NotFound.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ const defaultSearchArgs = {search_url: '/search'};
 /* eslint-disable sort-keys */
 const notFoundPages = {
   'area': {
-    title: N_l('Area Not Found'),
+    title: N_l('Area not found'),
```
Comment (yvanzo) [inline root/entity/NotFound.js:19]:
```suggestion
    title: N_lp('Area not found', 'header'),
```

**Review by mwiencek — APPROVED** [review]

File: root/utility/tableColumns.js [hunk 2]
```diff
@@ -747,7 +747,7 @@ export const relTypeColumn:
         {l_relationships(original.link_name)}
       </a>
     ),
-    Header: N_l('Relationship Type'),
+    Header: N_l('Relationship type'),
```
Comment (yvanzo) [inline root/utility/tableColumns.js:750]:
```suggestion
    Header: N_lp('Relationship type', 'header'),
```

File: root/utility/tableColumns.js [hunk 3]
```diff
@@ -762,7 +762,7 @@ export const seriesOrderingTypeColumn:
         ? lp_attributes(orderingType.name, 'series_ordering_type')
         : null;
     },
-    Header: N_l('Ordering Type'),
+    Header: N_l('Ordering type'),
```
Comment (yvanzo) [inline root/utility/tableColumns.js:765]:
```suggestion
    Header: N_lp('Ordering type', 'header'),
```

File: root/utility/tableColumns.js [hunk 4]
```diff
@@ -818,6 +818,6 @@ export const workLanguagesColumn:
         ))}
       </ul>
     ),
-    Header: N_l('Lyrics Languages'),
+    Header: N_l('Lyrics languages'),
```
Comment (yvanzo) [inline root/utility/tableColumns.js:821]:
```suggestion
    Header: N_lp('Lyrics languages', 'header'),
```

**Review by yvanzo — APPROVED** [review]
Only three missing occurrences, looks good otherwise.

**Pull request merged**